src/lib/eco-router/api.ts

```diff
@@ -269,6 +292,27 @@ export async function getExactOut(
       })
   })
 
+  const velodromeTrade = new Promise<any | undefined>(resolve => {
+    if (!RoutablePlatform.VELODROME.supportsChain(chainId)) {
+      return resolve(undefined)
+    }
+
+    VelodromeTrade.getQuote({
+      quoteCurrency: currencyIn,
+      amount: currencyAmountOut,
+      maximumSlippage,
+      recipient: receiver,
+      tradeType: TradeType.EXACT_OUTPUT,
```

Comment:
`tradeType` should determined by the implementation of `getQuote`. Something to do in the SDK

Reply:
On the sdk side we handle tradeType but it is determined from front-end....Like the rest of the trades.

Reply:
Totally lost context there. You're right. My bad.